### PR TITLE
Added :target to Page Inspector pseudo-class toggles

### PR DESCRIPTION
**Pull request by SebastianZ**
Updated pseudo-class toggle options of Page Inspector to include the `:target` pseudo-class. And also added it to the release notes for Firefox 87. See [bug 1689899](https://bugzilla.mozilla.org/show_bug.cgi?id=1689899).

This change is all mine, from filing the bug over creating the patch plus test to documenting it. 😄 _And_ it will be my first (and surely not last) contribution since the contents moved to GitHub. 🥳 

Sebastian

### CODE REVIEW

**Comment by hamishwillee** [comment]
@SebastianZ Fantastic! I'm working on Friday and will look at this then (assuming it is not reviewed by someone else first). FYI this is erroring because the images you added could be more compressed - see https://github.com/mdn/content/pull/2020/checks?check_run_id=1817070267#step:9:19

As it says, you need to run `yarn filecheck "files/en-us/tools/page_inspector/how_to/examine_and_edit_css/show_pseudo_classes_hover.png" --save-compression` for each of the files/.

This does not currently work (for me) on windows, but it does on Ubuntu.

**Comment by chrisdavidmills** [comment]
Hah, lovely to see you here @SebastianZ !

I am going to leave you in @hamishwillee 's capable hands for this one, but please shout if you folks need any assistance (e.g. I'm on macOS, so could apply the image compression to the build if needed).

**Comment by SebastianZ** [comment]
I am also on Windows and I didn't figure out yet how to get Yarn to run there. I'll see if I can get it to work somehow.

It would be great if there were more information about this than

> Note, this is how you do it on terminals with bash (macOS and Linux). You have to do it differently on Windows.

Just as a side note. 😄 

Sebastian

**Comment by SebastianZ** [comment]
Ok, got Yarn running by installing it via npm. As @hamishwillee already pointed out, the compression doesn't work though. I get an error that the command cannot be processed syntatically and it results in an exit code 255. 😞 I'll see what I can do.

Sebastian

**Comment by hamishwillee** [comment]
@SebastianSimon Hot of the press, there is a workaround for compression on windows. On the file in the content root named [package.json](https://github.com/mdn/content/blob/main/package.json#L12) change the line:
```
"filecheck": "env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/filecheck/cli.js --cwd=$(pwd)",
```
TO 
```
"filecheck": "env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/filecheck/cli.js --cwd=.",
```

(ie `--cwd=.`)

I'm back on this tomorrow.

**Comment by SebastianZ** [comment]
That was it! Thank you very much, @hamishwillee! Especially for helping while not at work!

Sebastian

PS: You mentioned the wrong Sebastian in your comment. 😄

**Comment by SebastianSimon** [comment]
> PS: You mentioned the wrong Sebastian in your comment. :smile:

Ah, that’s why I’m randomly receiving emails; I was so confused. :smile:

**Comment by hamishwillee** [comment]
Many apologies - who would have thought there could be two Sebastians in the world :-).

File: files/en-us/tools/page_inspector/how_to/examine_and_edit_css/index.html [hunk 1]
```diff
@@ -153,7 +153,7 @@ <h3 id="Viewing_common_pseudo-classes">Viewing common pseudo-classes</h3>
 
 <p><img alt="" src="show_pseudo_classes.png" style="border: 1px solid black; display: block; margin: 0 auto;"></p>
 
-<p>Click the button to see checkboxes that you can use to enable the {{cssxref(":hover")}}, {{cssxref(":active")}} and {{cssxref(":focus")}}, {{cssxref(":focus-within")}}, {{cssxref(":focus-visible")}} and {{cssxref(":visited")}} pseudo-classes for the selected element:</p>
+<p>Click the button to see checkboxes that you can use to enable the {{cssxref(":hover")}}, {{cssxref(":active")}} and {{cssxref(":focus")}}, {{cssxref(":focus-within")}}, {{cssxref(":focus-visible")}}, {{cssxref(":visited")}}, and {{cssxref(":target")}} pseudo-classes for the selected element:</p>
```
Comment (hamishwillee) [inline files/en-us/tools/page_inspector/how_to/examine_and_edit_css/index.html:156]: This is great, but it reflects FF87, which won't be out for more than a month (if this was FF86 I'd just merge it). So either we should hold this off merging, or modify it now to say something like:
> <p>Click the button to see checkboxes that you can use to enable the {{cssxref(":hover")}}, {{cssxref(":active")}} and {{cssxref(":focus")}}, {{cssxref(":focus-within")}}, {{cssxref(":focus-visible")}}, {{cssxref(":visited")}}, and {{cssxref(":target")}}  **(From Firefox 87)** pseudo-classes for the selected element:</p>

We can delete that FF87 after FF86 goes out. Thoughts?

**Comment by hamishwillee** [comment]
Looks great @SebastianZ . 

Please see inline comments. Could you also update the screenshot for the image **after** the one you updated - https://github.com/mdn/content/pull/2020/files#diff-23e77088e1e57c2827717e5b4491d2cdfca0614474fdfec364ea9012edf29c7cR162  - as this also shows the selector toggles.

**Comment by SebastianZ** [comment]
> 
> 
> Looks great @SebastianZ .
> 
> Please see inline comments. Could you also update the screenshot for the image **after** the one you updated - https://github.com/mdn/content/pull/2020/files#diff-23e77088e1e57c2827717e5b4491d2cdfca0614474fdfec364ea9012edf29c7cR162 - as this also shows the selector toggles.

Oh, right! How could I miss that? I've replaced that image now, too.

Sebastian

File: files/en-us/tools/page_inspector/how_to/examine_and_edit_css/index.html (hunk 1 above)
Comment (SebastianZ) [reply files/en-us/tools/page_inspector/how_to/examine_and_edit_css/index.html:156]: Placing the note after the list might get misinterpreted and we also might forget to remove the note later on.

I'm not in a hurry. 😄 So, if you think we should wait until 86 is out, I'm fine with that. 

Sebastian

**Comment by hamishwillee** [comment]
Great!

@chrisdavidmills This is good, but should not be merged until FF86 is in, and users can reasonably expect to start seeing docs for FF87. I have marked as "WIP - do not merge" in the title. Can you create an FF87 project and add this so we don't miss it?

**Comment by chrisdavidmills** [comment]
I'd be OK to merge this, as version 87 is available as Firefox Nightly, provided we say it is aailable in 87 onwards...

One thing though — I can't see it in my Nightly. Is it behind a flag or something? I just updated.

**Comment by Ryuno-Ki** [comment]
> I have marked as "WIP - do not merge" in the title.

Would it be possible to also mark it as a Draft PR? That would effectively block the merge in GitHub.

**Comment by hamishwillee** [comment]
@chrisdavidmills 

> version 87 is available as Firefox Nightly, provided we say it is aailable in 87 onwards...

We discussed that the text to add "from v87 onwards" looks pretty ugly here, and we'd need to add a second PR in a month when FF87 comes out to remove it. So we figured we'd rather just wait/not merge. 

@Ryuno-Ki I don't have any rights to revert things to draft, which is why I marked heading as WIP. If you do then please make the change.

**Comment by Ryuno-Ki** [comment]
It's a draft now. Let us know once you feel it's complete enough to get merged.
Current subscriptions by people who commented are still in place.

**Comment by chrisdavidmills** [comment]
> We discussed that the text to add "from v87 onwards" looks pretty ugly here, and we'd need to add a second PR in a month when FF87 comes out to remove it. So we figured we'd rather just wait/not merge.

Ah OK, makes sense. Thanks for the clarification.

**Comment by hamishwillee** [comment]
@chrisdavidmills This is now ready to go in as fix for #2508

**Review by chrisdavidmills — APPROVED** [review]
I already looked at this...let's get it merged ;-)

**Comment by SebastianZ** [comment]
I guess I can mark the related bug as `dev-doc-complete` now, right?

Sebastian

**Comment by chrisdavidmills** [comment]
> I guess I can mark the related bug as `dev-doc-complete` now, right?
> 
> Sebastian

yup! And provide a reference to this issue, so the engineers can see what work happened.

**Pull request merged**